util/SDIF/CMakeLists.txt

```diff
@@ -2,5 +2,5 @@
 
 if(BUILD_UTILITIES)
 set(sdif2ad_SRCS sdif2adsyn.c sdif.c sdif-mem.c)
-make_executable(sdif2ad "${sdif2ad_SRCS}" "")
+make_utility(sdif2ad "${sdif2ad_SRCS}" "")

```

Comment:
This should be a utility because it relies on csound I think?

Reply:
Whats the difference between an executable and a utility though? To me it seems the same

Reply:
Based on my reading, utilites rely on libcsound and executables don't. They aren't the most descriptive names though...

Reply:
Ok I looked at the code. I didn't realise `make_utility` is calling `make_executable`. Really not sure if there is much value in these anymore with cmake targets. Can worry about removing them later.